client/src/screens/DashboardScreen/DashboardScreen.tsx

```diff
@@ -1,23 +1,12 @@
-import Header from "../../components/Header/Header";
+import CertificationRow from "../../components/CertificationRow/CertificationRow";
+import DashboardTable from "../../components/DashboardTable/DashboardTable";
+import DashboardTableContainer from "../../components/DashboardTable/DashboardTableContainer";
 import "./DashboardScreen.css";
 
-const certificate = {
-  id: "1",
-  name: "SCRUM Master Certificate",
-  type: "IBM",
-  categories: [
-    { id: "1", name: "SCRUM" },
-    { id: "2", name: "Organization" },
-  ],
-  expiration_date: "Feb 14th, 2099",
-};
-
 const DashboardScreen = () => {
-  return (
-    <div className="background-screen">
-      <Header title={"Dashboard"} />
```

Comment:
Why did you delete Header line?

Reply:
Idk what happened, I don't remember deleting tbh, I think it was already gone when I pulled the dev branch. I tried adding it again but the header is not even a file anymore.
(UPDATE: It's here now lol I swear I couldn't find it yesterday at night)